packages/button/src/button.tsx

```diff
@@ -140,16 +140,27 @@ export const Button = forwardRef<ButtonProps, "button">((props, ref) => {
       {leftIcon && !isLoading && (
         <ButtonIcon marginEnd={iconSpacing}>{leftIcon}</ButtonIcon>
       )}
+
       {isLoading && spinnerPlacement === "start" && (
-        <ButtonSpinner label={loadingText} placement="start">
+        <ButtonSpinner
+          data-testid="placement-start"
```

Comment:
I'm a bit weary of adding `data-testid` to this component since it'll introduce a new pattern that needs to go in all packages. Could you consider using the `className` for the button spinner `chakra-button__spinner` ?

Reply:
Sure, i'll implement something like `chakra-button__spinner--start` and `chakra-button__end` to follows BEM, looks fine ?